coralproject/talk
# Permitted Domains for Talk Install

## What does this PR do?
- Adds `Permited Domains` to Talk Install
- Adds Translations for Install
- Adds new translations for `admin/configure`

## How do I test this PR?

- Drop the DB
- Go to `/admin/install`
- The third step should be `Add Permitted Domains`
